src/Angor/Client/Pages/Invest.razor

```diff
@@ -383,16 +383,16 @@
             feeData.FeeEstimations.Fees.AddRange(fetchFees);
             feeData.SelectedFeeEstimation = feeData.FeeEstimations.Fees.First();
 
-            var investorKey = _derivationOperations.DeriveInvestorKey(_walletStorage.GetWallet(), project.ProjectInfo.FounderKey);
+            var investorKey = _derivationOperations.DeriveInvestorKey(_walletStorage.GetWallet().WalletWords, project.ProjectInfo.FounderKey);
```

Comment:
We call get wallet 3 times in the same method, can you change it to one call?

Reply:
Good point in fact this can be a point if improving perf, we parse the wallet every time we pass the words, instead we could derive the parent key once and pass that down the method

Reply:
You mean keep it in the keys in the wallet?

Reply:
yes, or derive it once and pass that to the method, right now we derive it each time in all 3 cases.